<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WineYardsProgramEntry</class>
 <widget class="QWidget" name="WineYardsProgramEntry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>140</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>140</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>850</width>
    <height>140</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="KTitleWidget" name="programTitle">
     <property name="minimumSize">
      <size>
       <width>800</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Untitled Program</string>
     </property>
     <property name="comment">
      <string>A Windows program</string>
     </property>
     <property name="icon">
      <iconset theme="distributor-logo-windows">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="toolbar" native="true">
     <property name="minimumSize">
      <size>
       <width>800</width>
       <height>50</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#toolbar {
	background-color: #5e81ac;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="KLed" name="runningIndicator">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="state">
         <enum>KLed::Off</enum>
        </property>
        <property name="shape">
         <enum>KLed::Circular</enum>
        </property>
        <property name="look">
         <enum>KLed::Flat</enum>
        </property>
        <property name="color">
         <color>
          <red>94</red>
          <green>129</green>
          <blue>172</blue>
         </color>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KBusyIndicatorWidget" name="loadingIndicator">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="KSeparator" name="ind_separator">
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="midLineWidth">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="steamBtn">
        <property name="text">
         <string>Launch with Steam</string>
        </property>
        <property name="icon">
         <iconset theme="steam">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="runBtn">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset theme="media-playback-start">
          <normaloff>.</normaloff>.</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="stopBtn">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset theme="media-playback-stop">
          <normaloff>../../../../home/tridentur</normaloff>../../../../home/tridentur</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="terminalBtn">
        <property name="text">
         <string>Launch with Konsole</string>
        </property>
        <property name="icon">
         <iconset theme="utilities-terminal">
          <normaloff>../../../../home/tridentur</normaloff>../../../../home/tridentur</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="pathBtn">
        <property name="text">
         <string>Browse path</string>
        </property>
        <property name="icon">
         <iconset theme="document-open">
          <normaloff>../../../../home/tridentur</normaloff>../../../../home/tridentur</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KBusyIndicatorWidget</class>
   <extends>QWidget</extends>
   <header>kbusyindicatorwidget.h</header>
  </customwidget>
  <customwidget>
   <class>KLed</class>
   <extends>QWidget</extends>
   <header>kled.h</header>
  </customwidget>
  <customwidget>
   <class>KSeparator</class>
   <extends>QFrame</extends>
   <header>kseparator.h</header>
  </customwidget>
  <customwidget>
   <class>KTitleWidget</class>
   <extends>QWidget</extends>
   <header>ktitlewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
